File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p02613/p02613_num9_parsed.py
# Program

N = int(input())
verdicts = []
for i in range(N):
    verdicts.append(input())

counts = {
    "AC": 0,
    "WA": 0,
    "TLE": 0,
    "RE": 0
}

for verdict in verdicts:
    counts[verdict] += 1

print("AC x", counts["AC"])
print("WA x", counts["WA"])
print("TLE x", counts["TLE"])
print("RE x", counts["RE"])

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p02613/p02613_num9_parsed.cpp
#include <iostream>
#include <map>
#include <string>

int main() {
    int N;
    std::cin >> N;

    std::map<std::string, int> verdicts;
    verdicts["AC"] = 0;
    verdicts["WA"] = 0;
    verdicts["TLE"] = 0;
    verdicts["RE"] = 0;

    for (int i = 0; i < N; i++) {
        std::string s;
        std::cin >> s;
        verdicts[s]++;
    }

    std::cout << "AC x " << verdicts["AC"] << std::endl;
    std::cout << "WA x " << verdicts["WA"] << std::endl;
    std::cout << "TLE x " << verdicts["TLE"] << std::endl;
    std::cout << "RE x " << verdicts["RE"] << std::endl;

    return 0;
}